src/applications/avs/components/ItemsBlock.jsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { allFieldsEmpty } from '../utils';
+
 const ItemsBlock = props => {
   const { heading, intro, itemType, items, renderItem, showSeparators } = props;
 
-  if (!items || items.length === 0) {
-    return null;
-  }
+  if (!items) return null;
+
+  // Filter out null/empty field values.
+  let listItems = items.filter(item => !allFieldsEmpty(item));
```

Comment:
NITPICK: Is there any chance that items will not be an array?

Reply:
Items _shouldn't_ be anything other than an array, but it's a good point. I think what I'll do is make a separate PR to update ItemsBlock and the other formatting components to throw an error if they encounter a wrong type. (On the theory that it's better to bomb out than show something incorrect or not show data that should be there)